aten/src/ATen/native/transformers/sdp_utils_cpp.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
+#include <cstdint>
 namespace sdp {
+
+constexpr int32_t num_backends = 3;
```

Comment:
This seems like a disturbingly generic name even if it's within the sdp namespace.